pipelines/build-deploy-test-CI-WL.yml

```diff
@@ -1063,6 +1097,24 @@ templating:
 
 jobs:
 
+# Run DB Teardown
+- name: "Reset DB"
```

Comment:
Should be something like "Reset CI DB" and be in the CI section since this pipeline targets multiple environments